plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,217 @@
+<?php
+/**
+ * Woo AI plugin main class
+ *
+ * @package Woo_AI
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\Jetpack\Constants;
+
+/**
+ * Woo_AI Main Class.
```

Comment:
This seems to kinda be a duplicate of the above comment (the class name is incorrect as well):
```
/**
 * Woo AI plugin main class
 *
 * @package Woo_AI
 */
```

I'm personally okay with just omitting these types of docblock comments unless they have an explanation of what they accomplish.

Reply:
I get a linting error if I omit the heading, but I definitely need to update this 👍🏻 
![image](https://github.com/woocommerce/woocommerce/assets/444632/46520379-5b2c-44e4-a398-928ea14a56ec)